playgrounds/cra/package.json

```diff
@@ -12,6 +12,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "test:typescript": "exit 0",
```

Comment:
this one is also somewhat interesting, I don't mind this but I don't quite see why this has been added here as the setup of the CI scripts etc has not been touched

Reply:
Without this `yarn test:typescript` in the workspace root will fail. This command is not actually run by the GitHub workflows which is why it was not noticed sooner.

In Yarn 2, they made it so `yarn workspaces run X` only runs the script X in packages that actually define it.